src/App.js

```diff
@@ -1,7 +1,29 @@
+import { Route, Routes } from "react-router-dom";
 import "./App.css";
+import { ExplorePage } from "./pages/explore/explore";
+import { HistoryPage } from "./pages/history/history";
+import { HomePage } from "./pages/Homepage/pages";
+import { LikedPage } from "./pages/liked/liked";
+import { LogInPage } from "./pages/Login/login";
+import { PlaylistPage } from "./pages/playlist/playlist";
+import { SignUpPage } from "./pages/signUp/signUp";
+import { WatchLaterPage } from "./pages/watchLater/watchLater";
```

Comment:
Suggestion :  Add a `index.js` in `pages folder` and export everything from there
eg. `export {ExplorePage} from "./explore/explore"`

So, what will happen when you'll import the pages it will look like
`import {ExplorePage, HistoryPage,...,WatchLaterPage} from "./pages"`

This will reduce lines of import

Reply:
okay I will update in next pr